requirements-dev.txt

```diff
@@ -0,0 +1,32 @@
+cfgv==3.4.0
```

Comment:
Is it required to have these files in the repo root? If no, I would create a `dev/` directory.